pybaseball/utils.py

```diff
@@ -110,3 +111,61 @@ def get_text_file(url):
         s = f.text
 
     return s
+
+def flag_imputed_data(statcast_df):
+    """Function to flag possibly imputed data as a result of no-nulls approach (see: https://tht.fangraphs.com/43416-2/)
+       Note that this imputation only occured with TrackMan, not present in Hawk-Eye data (beyond 2020)
+    Args:
+        statcast_df (pd.DataFrame): Dataframe loaded via statcast.py, statcast_batter.py, or statcast_pitcher.py
+    Returns:
+        pd.DataFrame: Copy of original dataframe with "possible_imputation" flag
+    """
+
+    ParameterSet = namedtuple('ParameterSet', ["ev", "angle", "bb_type"])
+    impute_combinations = []
+
+    # pop-ups
+    impute_combinations.append(ParameterSet(ev=80, angle=69, bb_type="popup"))
+    impute_combinations.append(ParameterSet(ev=37, angle=62, bb_type="popup"))
+    impute_combinations.append(ParameterSet(ev=86, angle=67, bb_type="popup"))
+
+    # Flyout
+    impute_combinations.append(ParameterSet(ev=71.4, angle=36.0, bb_type="fly_ball"))
+    impute_combinations.append(ParameterSet(ev=89, angle=39, bb_type="fly_ball"))
+    impute_combinations.append(ParameterSet(ev=89.2, angle=39.3, bb_type="fly_ball"))
```

Comment:
when I look at the statcast data, I see a lot of values at `ev=89.2`, `angle=39.0`. and not `ev=89.2`, `angle=`39.3`. I'm wondering if this record with `angle=39.3` might be a typo?

Reply:
in fact it looks like maybe all the imputed values have angle that are integers?

Reply:
So that originally came from the tables found [here](https://tht.fangraphs.com/43416-2/), under "Stringer Fly Balls," then rounded to the precision in the DFs we get. I'll do some more investigating though to see if something was missed.

Reply:
it looks like they've started rounding all angles to integer?

Reply:
Ah yeah - so I guess it varies by year... pains :( Maybe this is a product of HawkEye? As I commented below, probably your thought of using a heuristic derivation is the right move.